.github/workflows/preview.yml

```diff
@@ -66,14 +55,18 @@ jobs:
 
       - name: Create preview layout
         run: |
-          mkdir -p preview/${{ fromJSON(env.PR_DATA).target_name }}
-          unzip -d preview foreman-docs-web-master.zip
-          unzip -d preview/${{ fromJSON(env.PR_DATA).target_name }} foreman-docs-html-${{ fromJSON(env.PR_DATA).branch_name }}.zip
+          gh run download '${{ github.event.worfklow_run.id }}' --name 'foreman-docs-web-master' --dir preview
```

Comment:
Due to a merge conflict, Worf is back. Surprisingly, https://github.com/theforeman/foreman-documentation/actions/runs/12831182509/job/35780996640 passed:
> gh run download '' --name 'foreman-docs-web-master' --dir preview

The preview and diff look correct so perhaps the workflow ID is not needed at all and it's smart enough to detect it?

Reply:
Answering my own question from `man gh-run-download`:
> By default, this command downloads the latest artifact created and uploaded through GitHub Actions. Because workflows can  delete  or  overwrite  artifacts, <run-id> must be used to select an artifact from a specific workflow run.

So it worked because it was the last run, but that's not guaranteed to be correct. https://github.com/theforeman/foreman-documentation/pull/3593 should address this.

Overall I think this is now almost done, even though it was bumpy.

Reply:
Hmm, this is a good sign to me: https://github.com/theforeman/foreman-documentation/pull/3592#issuecomment-2598537737

So yes, we should either fix the typo and ignore that it's not really necessary or drop it?